Source/Core/OrientedBoundingBox.js

```diff
@@ -606,25 +606,24 @@ define([
         return result;
     };
 
-    var scratchBoundingSphere = new BoundingSphere();
-
     /**
-     * Determines whether or not a bounding box is hidden from view by the occluder.
+     * Creates a bounding sphere that contains the oriented bounding box.
      *
-     * @param {OrientedBoundingBox} sphere The bounding box surrounding the occludee object.
-     * @param {Occluder} occluder The occluder.
-     * @returns {Boolean} <code>true</code> if the sphere is not visible; otherwise <code>false</code>.
+     * @param {OrientedBoundingBox} box The bounding box.
+     * @param {BoundingSphere} [result] The bounding sphere onto which to store the result.
+     * @returns {BoundingSphere} The computed bounding sphere.
      */
-    OrientedBoundingBox.isOccluded = function(box, occluder) {
+    OrientedBoundingBox.toBoundingSphere = function(box, result) {
```

Comment:
Use `BoundingSphere.fromCornerPoints` or add a trivial `BoundingSphere.fromOrientedBoundingBox` (including tests, doc, and CHANGES.md), which could be a separate pull request into this branch and into master.

In general, we use `from` functions, not `to`.